src/weatherService.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/no-unresolved */
+import axios from 'axios';
+
+const processResponse = (res) => ({
+  city: res.data.name,
+  weatherDesc: res.data.weather[0],
+  temp: res.data.main.temp,
+  pressure: res.data.main.pressure,
+  humidity: res.data.main.humidity,
+  visibility: (res.data.visibility ? res.data.visibility : 'Not available'),
+  wind: res.data.wind,
+  cloud: res.data.clouds.all,
+  rain: (res.data.rain ? res.data.rain : undefined),
+  snow: (res.data.snow ? res.data.snow : undefined),
+});
+
+const getWeather = (city, state, country) => {
+  let searchString = `${city.trim().split(' ').join('+')},`;
+  if (state.trim() !== '') {
+    searchString += `${state ? state.trim().split(' ').join('+') : ''},`;
+  }
+  searchString += `${country.trim()}`;
+
+  const apiKey = '5d11c0834bf383929d17b0a9c78b7214';
+  return axios.get(`https://api.openweathermap.org/data/2.5/weather?q=${searchString}&appid=${apiKey}`)
```

Comment:
Let's look into the project specification, as we both know, all other students are using this too;
so, we need to;
1. make different requests for the Celsius data and the Fahrenheit data.

![image](https://user-images.githubusercontent.com/31897434/81986015-06cdc780-962f-11ea-9fe5-60a32764c8db.png)